website/redirects.js

```diff
@@ -0,0 +1,1200 @@
+    destination:
+      'https://learn.hashicorp.com/collections/nomad/job-failure-handling',
+    permanent: true,
+  },
+  {
+    source:
+      '/docs/operating-a-job/failure-handling-strategies/reschedule(.html)?',
+    destination:
+      'https://learn.hashicorp.com/tutorials/nomad/failures-reschedule',
+    permanent: true,
+  },
+  {
+    source: '/docs/operating-a-job/failure-handling-strategies/restart(.html)?',
+    destination: 'https://learn.hashicorp.com/tutorials/nomad/failures-restart',
+    permanent: true,
+  },
+  {
+    source: '/docs/operating-a-job/index.html',
+    destination: 'https://learn.hashicorp.com/collections/nomad/manage-jobs',
+    permanent: true,
+  },
+  {
+    source: '/docs/operating-a-job/inspecting-state.html',
+    destination: 'https://learn.hashicorp.com/tutorials/nomad/jobs-inspect',
+    permanent: true,
+  },
+  {
+    source: '/docs/operating-a-job/resource-utilization.html',
+    destination: 'https://learn.hashicorp.com/tutorials/nomad/jobs-utilization',
+    permanent: true,
+  },
+  {
+    source: '/docs/operating-a-job/submitting-jobs.html',
+    destination: 'https://learn.hashicorp.com/tutorials/nomad/jobs-submit',
+    permanent: true,
+  },
+  {
+    source: '/docs/operating-a-job/update-strategies',
+    destination: 'https://learn.hashicorp.com/collections/nomad/job-updates',
+    permanent: true,
+  },
+  {
+    source:
+      '/docs/operating-a-job/update-strategies/blue-green-and-canary-deployments(.html)?',
+    destination:
+      'https://learn.hashicorp.com/tutorials/nomad/job-blue-green-and-canary-deployments',
+    permanent: true,
+  },
+  {
+    source: '/docs/operating-a-job/update-strategies/handling-signals(.html)?',
+    destination:
+      'https://learn.hashicorp.com/tutorials/nomad/job-update-handle-signals',
+    permanent: true,
+  },
+  {
+    source: '/docs/operating-a-job/update-strategies/index.html',
+    destination: 'https://learn.hashicorp.com/collections/nomad/job-updates',
+    permanent: true,
+  },
+  {
+    source: '/docs/operating-a-job/update-strategies/rolling-upgrades(.html)?',
+    destination:
+      'https://learn.hashicorp.com/tutorials/nomad/job-rolling-update',
+    permanent: true,
+  },
+  {
+    source: '/docs/runtime/schedulers(.html)?',
+    destination: '/docs/schedulers',
+    permanent: true,
+  },
+  {
+    source: '/docs/service-discovery/(index.html)?',
+    destination: '/docs/integrations/consul-integration',
+    permanent: true,
+  },
+  {
+    source: '/docs/telemetry/overview',
+    destination: '/docs/telemetry',
+    permanent: true,
+  },
+  {
+    source: '/docs/vault-integration',
+    destination: '/docs/integrations/vault-integration',
+    permanent: true,
+  },
+  {
+    source: '/guides',
+    destination: 'https://learn.hashicorp.com/nomad',
+    permanent: true,
+  },
+  {
+    source: '/guides/:splat*',
+    destination: 'https://learn.hashicorp.com/nomad',
```

Comment:
Ah, nice catch on this one. It looks like this route was intended as a catch-all for `/guides/*`, but since we've sorted alphabetically, it doesn't behave the same way. Looks like its probably safest to stick with the Netlify ordering, and try to clean things up later on a more case-by-case basis.